### PR TITLE
(fix) O3-2852: Fix the viewport size of the Start Visit form on tablet 

### PR DESCRIPTION
**Pull request by Madhu-mac**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Fixes the Start Visit form UI on the tablet viewport so that the form covers the entire vertical height of the viewport. 

### Before
 
Screenshot_20240330_032237

### After
 
tablet

## Related issue

https://openmrs.atlassian.net/issues/O3-2852

### CODE REVIEW

**Comment by Madhu-mac** [comment]
@njiddasalifu I Fixed this error. Can you review it..

**Comment by Vijaykv5** [comment]
@Madhu-mac Kindly Add the ticket link associated with the issue.

**Comment by Madhu-mac** [comment]
@Vijaykv5 
**Here's the link:** https://openmrs.atlassian.net/issues/O3-2852?jql=%28text%20~%20%22O3-2852%22%20OR%20issuekey%20%3D%20%22O3-2852%22%29%20ORDER%20BY%20created%20DESC

**Comment by Vijaykv5** [comment]
> @Vijaykv5 **Here's the link:** https://openmrs.atlassian.net/issues/O3-2852?jql=%28text%20~%20%22O3-2852%22%20OR%20issuekey%20%3D%20%22O3-2852%22%29%20ORDER%20BY%20created%20DESC

@Madhu-mac I see that this issue has already been assigned to someone else. Please try to work on unassigned issue and I would suggest you to read [documentation](https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477483/Contribute+as+a+Developer#:~:text=To%20work%20on%20an%20issue,and%20make%20your%20first%20contribution.)

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss [hunk 1]
```diff
@@ -46,7 +45,7 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  height: calc(100vh - 6rem);
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss:49]: @Madhu-mac changing the height to `100vh` breaks the UI on the desktop viewport. The correct values are:

- var(--desktop-workspace-window-height) for desktop
- var(--tablet-workspace-window-height) for tablet

**Review by denniskigen — APPROVED** [review]
Thanks for taking a stab at this, @Madhu-mac.

**Pull request merged**